### DISCUSSION TITLE
Menu反应超级慢

### DISCUSSION THREAD

**Discussion by CaiWenlie** [vueComponent/ant-design-vue#4659] [Q&A] [answered]
Menu反应真的超慢，subitem越多越慢，我这里只放了十几个subitem，点击展开的时候就要卡顿1s了，大家有没有遇到一样的情况

![image](https://user-images.githubusercontent.com/8305535/133407095-ec9e3334-b4e9-44bb-9cf4-9ddc73b8e7fe.png)

![image](https://user-images.githubusercontent.com/8305535/133406945-f34375e4-46d8-4e1f-b086-600e7ceefbc4.png)

**Comment by ajuner** [comment]
是哪个版本

**Reply by CaiWenlie** [reply]
2.2.6

**Comment by sendya** [comment, marked as answer]
不要使用 vue-devtools
你现在把 vue-devtools 禁用，重开浏览器 试试感觉会不会卡

**Reply by CaiWenlie** [reply]
哎，真的是诶

**Reply by sendya** [reply]
vue3 版本的 devtools 分析性能的时候，会导致性能巨差，我目前不建议使用 vue3 版本的 devtools， 待 devtools 发正式版把
